src/app/workspace/[id]/page.jsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+import React from 'react'
+import { useRouter } from 'next/navigation'
+import ConversationCards from "@/components/ConversationCard/ConversationCard";
+
+const WorkSpaceIcon=({channelId,channelIcon,isActive})=>
+{
+    const router = useRouter();
+    return (
+        <li className={`pl-1 pr-2 py-1.5 ${isActive?"bg-white":"bg-transparent"} rounded-l-lg`} onClick={()=>router.push(`/workspace/${channelId}`)}>
+            <img src={channelIcon} width={48} height={48} className='rounded-lg' />
```

Comment:
we are using Nextjs, search Image Nextjs on google and add it

Reply:
Yes , Image component of next.js is used when we are rendering the page in server side in next.js

However this page uses state logic, so this has to be a "use client" page. 

Images must be stored as assets in next.js, since we are using a dummy link, have used the og <img> tag for easy n clean client side rendering of images 

Moreover it was giving me error when directly using Image component. You can branch n try it yourself to reproduce the error.

Reply:
Image can be used on client component

You mean this error?
![image](https://github.com/opencodeiiita/Hitch-Frontend/assets/77514129/d38bfd0d-0ea0-4c3d-9db7-52e783eebcd5)

add this to the next.config.js
`const nextConfig = {
  images: {
    remotePatterns: [
      {
        protocol: "https",
        hostname: '"encrypted-tbn0.gstatic.com"',
        port: "",
        pathname: "**",
      },
    ],
  },
};`